dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -616,44 +624,49 @@ private void createXMLFiles() throws ServletException, IOException, DotDataExcep
 				/* we will only export 10,000,000 items of any given type */
 				for(i=0;i < 10000000;i=i+step){
 
-                    _dh = new HibernateUtil(clazz);
-                    _dh.setFirstResult(i);
-                    _dh.setMaxResults(step);
+					_dh = new HibernateUtil(clazz);
+					_dh.setFirstResult(i);
+					_dh.setMaxResults(step);
 
                     //This line was previously like;
                     //_dh.setQuery("from " + clazz.getName() + " order by 1,2");
                     //This caused a problem when the database is Oracle because Oracle causes problems when the results are ordered
                     //by an NCLOB field. In the case of dot_containers table, the second field, CODE, is an NCLOB field. Because of this,
                     //ordering is done only on the first field for the tables, which is INODE
                     if(com.dotmarketing.beans.Tree.class.equals(clazz)){
-                    	_dh.setQuery("from " + clazz.getName() + " order by parent, child, relation_type");
+						_dh.setQuery("from " + clazz.getName() + " order by parent, child, relation_type");
                     }
                     else if(MultiTree.class.equals(clazz)){
-                    	_dh.setQuery("from " + clazz.getName() + " order by parent1, parent2, child, relation_type");
+						_dh.setQuery("from " + clazz.getName() + " order by parent1, parent2, child, relation_type");
                     }
                     else if(TagInode.class.equals(clazz)){
-                    	_dh.setQuery("from " + clazz.getName() + " order by inode, tag_id");
+						_dh.setQuery("from " + clazz.getName() + " order by inode, tag_id");
                     }
                     else if(Tag.class.equals(clazz)){
-                    	_dh.setQuery("from " + clazz.getName() + " order by tag_id, tagname");
+						_dh.setQuery("from " + clazz.getName() + " order by tag_id, tagname");
                     }
                     else if(CalendarReminder.class.equals(clazz)){
-                    	_dh.setQuery("from " + clazz.getName() + " order by user_id, event_id, send_date");
+						_dh.setQuery("from " + clazz.getName() + " order by user_id, event_id, send_date");
                     } 
                     else if(Identifier.class.equals(clazz)){
-                    	_dh.setQuery("from " + clazz.getName() + " order by parent_path, id");
-                    } else {
-                    	_dh.setQuery("from " + clazz.getName() + " order by 1");
+						dc = new DotConnect();
+						dc.setSQL("select * from identifier order by parent_path, id")
+								.setStartRow(i).setMaxRows(step);
+					} else {
+						_dh.setQuery("from " + clazz.getName() + " order by 1");
                     }
 
-                    _list = _dh.list();
+					if(Identifier.class.equals(clazz)){

```

Comment:
_list = (Identifier.class.equals(clazz))? .. : ...;

However it might be good to think in an idea to make it more generic in case more entities goes to SQL instead of Hibernate